ts/src/apex.ts

```diff
@@ -0,0 +1,1940 @@
+                'fee': '0',
+                'nonce': nonce,
+                'timestampSeconds': timestampSeconds,
+            };
+            const signature = await this.getZKTransferSignatureObj (this.safeString (this.options, 'seeds').replace ('0x', ''), orderToSign);
+            const request: Dict = {
+                'amount': amount.toString (),
+                'timestamp': timestampSeconds,
+                'clientTransferId': clientOrderId,
+                'signature': signature,
+                'zkAccountId': zkAccountId,
+                'subAccountId': subAccountId,
+                'fee': '0',
+                'token': code,
+                'tokenId': tokenId,
+                'receiverAccountId': receiverAccountId,
+                'receiverZkAccountId': receiverZkAccountId,
+                'receiverSubAccountId': receiverSubAccountId,
+                'receiverAddress': receiverAddress,
+                'nonce': nonce,
+            };
+            const response = await this.privatePostV3TransferOut (this.extend (request, params));
+            const data = this.safeDict (response, 'data', {});
+            const currentTime = this.milliseconds ();
+            return this.extend (this.parseTransfer (data, this.currency (code)), {
+                'timestamp': currentTime,
+                'datetime': this.iso8601 (currentTime),
+                'amount': this.parseNumber (amount),
+                'fromAccount': 'spot',
+                'toAccount': 'contract',
+            });
+        }
+    }
+
+    parseTransfer (transfer: Dict, currency: Currency = undefined): TransferEntry {
+        const currencyId = this.safeString (transfer, 'coin');
+        const timestamp = this.safeInteger (transfer, 'timestamp');
+        const fromAccount = this.safeString (transfer, 'fromAccount');
+        const toAccount = this.safeString (transfer, 'toAccount');
+        return {
+            'info': transfer,
+            'id': this.safeStringN (transfer, [ 'transferId', 'id' ]),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'currency': this.safeCurrencyCode (currencyId, currency),
+            'amount': this.safeNumber (transfer, 'amount'),
+            'fromAccount': fromAccount,
+            'toAccount': toAccount,
+            'status': this.safeString (transfer, 'status'),
+        };
+    }
+
+    /**
+     * @method
+     * @name apex#cancelAllOrders
+     * @description cancel all open orders in a market
+     * @see https://api-docs.pro.apex.exchange/#privateapi-v3-for-omni-post-cancel-all-open-orders
+     * @param {string} symbol unified market symbol of the market to cancel orders in
+     * @param {object} [params] extra parameters specific to the exchange API endpoint
+     * @returns {object[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+     */
+    async cancelAllOrders (symbol: Str = undefined, params = {}) {
```

Comment:
params should always be extended with the request upon calling the endpoint, like

```
 await this.privatePostV3DeleteOpenOrders (this.extend(request, params));
```